src/ncp-spinel/SpinelNCPControlInterface.cpp

```diff
@@ -438,6 +439,20 @@ SpinelNCPControlInterface::get_name() {
 	return mNCPInstance->get_name();
 }
 
+void
+SpinelNCPControlInterface::mfg(
+    const std::string& mfg_command,
+    CallbackWithStatusArg1 cb
+) {
+	mNCPInstance->start_new_task(boost::shared_ptr<SpinelNCPTask>(
+		new SpinelNCPTaskMfg(
```

Comment:
I have removed `SpinelNCPTaskMfg`, and use `SpinelNCPTaskSendCommand` instead.
My original idea is to make `mfg` a separated feature as in previous v0 implementation, but it seems unnecessary now.